sample-functions/AlpineFunction/Dockerfile

```diff
@@ -8,5 +8,8 @@ RUN chmod +x /usr/bin/fwatchdog
 # Populate example here
 # ENV fprocess="wc -l"
 
+RUN addgroup -g 1000 -S faas && adduser -u 1000 -S faas -G faas
```

Comment:
Please can you use the style from the other functions where we use the username `app`?

This is for consistency between [all templates and samples](https://github.com/openfaas/templates/blob/master/template/go/Dockerfile).

Reply:
I changed the user to be app, but I leave the USER <userid> to make it compliant with [OpenShift](https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines)

We might have to modify templates too